Main.bb

```diff
@@ -2151,7 +2151,7 @@ Function UpdateDoors()
 					EndIf
 					If d\AutoClose And RemoteDoorOn = True Then
 						If EntityDistance(Camera, d\obj) < 2.1 Then
-							If (Not Wearing714) Then PlaySound_Strict HorrorSFX(7)
+							If (Not Wearing714) Then PlaySound_Strict HorrorSFX(6)
```

Comment:
Horror7.ogg wasn't renamed to Horror6.ogg so this shouldn't have been changed.

Reply:
Crap